Modules/xASL_module_ASL.m

```diff
@@ -150,15 +151,16 @@
 %% D4. TimeEncoded parsing
 % Check if TimeEncoded is defined
 if isfield(x.Q,'TimeEncodedMatrixType') || isfield(x.Q,'TimeEncodedMatrixSize') 
-    x.modules.asl.bTimeEncoded = 1;
+    x.modules.asl.bTimeEncoded = true;
```

Comment:
We should add a warning here:

>> warning('We detected TimeEncoded data (e.g., Hadamard or Walsh), we will process this but this is still an experimental feature');

And similarly for multiPLD in this admin section:

>> warning('We detected multiPLD data, we will process this but this is still an experimental feature');

Reply:
ccec3a5

Reply:
a67a7bd